CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+# Contributing guide
+
+**First off all, thank you for taking the time to contribute into Prospero!** The below contents will help you through the steps for getting started with Prospero. Please make sure to read the relevant section before making your contribution. It will make it a lot easier for us maintainers and smooth out the experience for all involved. The community looks forward to your contributions.
+
+## Documentation
+
+You can find all information and examples in [Prospero documentation](https://docs.wildfly.org/prospero/)
+
+## Development environment
+
+* Latest Maven build tool https://maven.apache.org/
+* JDK 17+ You can use any distribution
+* [Eclipse](https://www.eclipse.org/downloads/packages/) or any other IDE allowing correct source formatting (IntelliJ)
+
+### Maven
+---
+
+#### Build prospero
+
+```
+    cd <PROSPERO_HOME>
+    mvn clean install
+```
+
+#### Building distribution
+
+The full distribution of Prospero includes a Wildfly feature pack, a standalone zip and documentation. Building those projects is excluded by default and enabled only if a maven `dist` profile is enabled.
```

Comment:
s/Wildfly feature pack/Galleon feature pack/

I think Galleon is the tech that provides the feature pack concept.